integration/test.ssr.ts

```diff
@@ -0,0 +1,23 @@
+import { exit } from 'shelljs';
+const request = require('./request-service');
+
+setTimeout(async () => {
```

Comment:
could you redo this code to use mocha?
https://www.npmjs.com/package/ts-mocha

```ts
describe('NGXS + SSR`, () => {
  it('Should be exist "NgxsOnInit todo"', () => {
    expect(body.include('NgxsOnInit todo')).toBe(true);
  });
  // ...
});
```

Reply:
No